### ISSUE TITLE
console: cannot select columns in insert/update permissions 

### ISSUE THREAD

**Issue by rikinsk** [hasura/graphql-engine#8935]
### Version Information

Server Version: v2.11

### Environment

All

### What is the current behaviour?

Cannot select individual columns in insert/update permissions. Only toggle all works

### What is the expected behaviour?

Allow selecting individual columns in insert/update permissions

### How to reproduce the issue?

1. Create a table
2. Go to permissions tab
3. Set row permissions for inset/update
4. Try selecting individual columns in the column permissions section

### Please provide any traces or logs that could help here.

```
vendor.js.gz:1 Uncaught TypeError: Cannot set properties of undefined (setting 'query_root_fields')
    at vendor.js.gz:1:5165715
    at produce (vendor.js.gz:1:4392706)
    at e (vendor.js.gz:1:5165661)
    at vendor.js.gz:1:5429218
    at produce (vendor.js.gz:1:4392706)
    at e (vendor.js.gz:1:5428879)
    at e (vendor.js.gz:1:4476433)
    at e (vendor.js.gz:1:8706307)
    at g (vendor.js.gz:1:8704198)
    at vendor.js.gz:1:25986246
 ```

**Comment by ilovett** [comment]
@rikinsk might this have to do with naming conventions or upgrading from a previous version?

**Comment by rikinsk** [comment]
Closed via ff65de1df4b36468158da040570e18ebf2053a62
